api/src/middleware/publications.js

```diff
@@ -4,7 +4,7 @@
 
 const { body, validationResult } = require('express-validator');
 const axios = require('axios');
-const categoryTypeEnum = require('../config/puppeteer');
+const { categoryTypeEnum } = require('../config/puppeteer');
```

Comment:
Part of refactoring because the validation for creating a publication wasn't working properly